content/rs/references/compatibility/resp.md

```diff
@@ -0,0 +1,82 @@
+---
+Title: RESP compatibility with Redis Enterprise
+linkTitle: RESP
+description: Redis Enterprise supports RESP2 and RESP3. 
+weight: 80
+alwaysopen: false
+toc: "true"
+categories: ["RS"]
+aliases: 
+---
+
+RESP (REdis Serialization Protocol) is the protocol that clients use to communicate with Redis databases. See the [RESP protocol specification](https://redis.io/docs/reference/protocol-spec/) for more information.
+
+## Supported RESP versions
+
+- RESP2 is supported by Redis Enterprise 6.0 and later.
```

Comment:
RESP2 was introduced in Redis 2.0, released in 2010. We should correct this statement to something like:
"All Redis Enterprise versions support RESP2"